bin/apisix

```diff
@@ -248,6 +248,22 @@ local function read_yaml_conf()
     return yaml.parse(ymal_conf)
 end
 
+local function checkOpenrestyVersion(need_version)
+	local shell="need_ver="..need_version ..";"
```

Comment:
style: 
1. bad indentation
2. when we connect a new string, a space is required between the objects: should be `"need_ver=" .. need_version  .. ";"`